packages/lesswrong/lib/crud/withMulti.ts

```diff
@@ -294,7 +294,10 @@ export function useMulti<
   const { query: locationQuery, location } = useLocation();
   const { history } = useNavigation();
 
-  let defaultLimit: number = (((locationQuery && queryLimitName && parseInt(locationQuery[queryLimitName])) || terms.limit) ?? initialLimit)
+  const locationQueryLimit = locationQuery && queryLimitName && !isNaN(parseInt(locationQuery[queryLimitName])) ? parseInt(locationQuery[queryLimitName]) : undefined;
+  const termsLimit = terms?.limit;
+  const defaultLimit: number = locationQueryLimit ?? termsLimit ?? initialLimit
```

Comment:
The cause of the bug was this diff from `15c9d2fcff3b52de24accd3277d3fbdb87ca5cdf` to `5b7c22ed6d11b161a4edd7c31e3ec73de22818d9`
```
-  const defaultLimit = ((locationQuery && queryLimitName && parseInt(locationQuery[queryLimitName])) || (terms && terms.limit) || initialLimit)
+  let defaultLimit: number = (((locationQuery && queryLimitName && parseInt(locationQuery[queryLimitName])) || terms.limit) ?? initialLimit)
```

The only thing actually required to fix it was to change `terms.limit` to `terms?.limit`, but I've also tried to make it clearer what it's doing (there was also an edge case where `defaultLimit` could end up as NaN, which didn't actually cause any problems, but I think it's better to fall back to `initialLimit` in that case)